src/ripple/net/impl/RPCCall.cpp

```diff
@@ -669,7 +670,21 @@ class RPCParser
     {
         std::string const strPk = jvParams[0u].asString ();
 
-        if (!parseBase58<PublicKey> (TokenType::TOKEN_ACCOUNT_PUBLIC, strPk))
+        bool const validPublicKey = [&strPk]{
+            if (parseBase58<PublicKey> (TokenType::TOKEN_ACCOUNT_PUBLIC, strPk))
+                return true;
+
+            std::pair<Blob, bool> pkHex(strUnHex (strPk));
+            if (!pkHex.second)
+                return false;
```

Comment:
Suggestion: This code path is not covered by unit test. We can improve the coverage and test by testing with a non-hexidecimal key.